openbas-front/tests_e2e/model/contracts/contract-api.ts

```diff
@@ -0,0 +1,360 @@
+import { Page } from '@playwright/test';
+
+class ContractApiMock {
+  constructor(private page: Page) {
+  }
+
+  async mockContracts() {
```

Comment:
I don't undestand why we need to mock the back-end in e2e tests

Reply:
Because the response from back can change so I prefer test with stable data :)

Reply:
After reflection, I think we need to dig into why the data changes rather than mocking the back-end.
Otherwise, I have the feeling that it becomes a front-end test and not e2e